floris/core/farm.py

```diff
@@ -134,21 +138,29 @@ def __attrs_post_init__(self) -> None:
         # This allows to read the yaml input files once rather than every time they're given.
         # In other words, if the turbine type is already in the cache, skip that iteration of
         # the for-loop.
+
         turbine_definition_cache = {}
```

Comment:
Could this just be modified to be `self._turbine_definition_cache`? Then you could remove line 219, I believe. Also, update `turbine_definition_cache` to `self._turbine_definition_cache` throughout.